projects/schematics/src/migrations/5_0/rename-symbol/cart-lib-checkout.migration.ts

```diff
@@ -0,0 +1,1354 @@
+    previousImportPath: '@spartacus/storefront',
+    newImportPath: '@spartacus/order/components',
+  },
+  {
+    previousNode: 'ReplenishmentOrderDetailsService',
+    previousImportPath: '@spartacus/storefront',
+    newImportPath: '@spartacus/order/components',
+  },
+  {
+    previousNode: 'ReplenishmentOrderHistoryComponent',
+    previousImportPath: '@spartacus/storefront',
+    newImportPath: '@spartacus/order/components',
+  },
+  {
+    previousNode: 'ReplenishmentOrderHistoryModule',
+    previousImportPath: '@spartacus/storefront',
+    newImportPath: '@spartacus/order/components',
+  },
+  {
+    previousNode: 'ReturnOrderComponent',
+    previousImportPath: '@spartacus/storefront',
+    newImportPath: '@spartacus/order/components',
+  },
+  {
+    previousNode: 'ReturnOrderConfirmationComponent',
+    previousImportPath: '@spartacus/storefront',
+    newImportPath: '@spartacus/order/components',
+  },
+  {
+    previousNode: 'ReturnOrderConfirmationModule',
+    previousImportPath: '@spartacus/storefront',
+    newImportPath: '@spartacus/order/components',
+  },
+  {
+    previousNode: 'ReturnOrderModule',
+    previousImportPath: '@spartacus/storefront',
+    newImportPath: '@spartacus/order/components',
+  },
+  {
+    previousNode: 'ReturnRequestDetailModule',
+    previousImportPath: '@spartacus/storefront',
+    newImportPath: '@spartacus/order/components',
+  },
+  {
+    previousNode: 'ReturnRequestItemsComponent',
+    previousImportPath: '@spartacus/storefront',
+    newImportPath: '@spartacus/order/components',
+  },
+  {
+    previousNode: 'ReturnRequestListModule',
+    previousImportPath: '@spartacus/storefront',
+    newImportPath: '@spartacus/order/components',
+  },
+  {
+    previousNode: 'ReturnRequestOverviewComponent',
+    previousImportPath: '@spartacus/storefront',
+    newImportPath: '@spartacus/order/components',
+  },
+  {
+    previousNode: 'ReturnRequestTotalsComponent',
+    previousImportPath: '@spartacus/storefront',
+    newImportPath: '@spartacus/order/components',
+  },
+  {
+    previousNode: 'SaveForLaterComponent',
+    previousImportPath: '@spartacus/storefront',
+    newImportPath: '@spartacus/cart/base/components',
+  },
+  {
+    previousNode: 'SaveForLaterModule',
+    previousImportPath: '@spartacus/storefront',
+    newImportPath: '@spartacus/cart/base/components',
+  },
+  {
+    previousNode: 'TrackingEventsComponent',
+    previousImportPath: '@spartacus/storefront',
+    newImportPath: '@spartacus/order/components',
+  },
+  {
+    previousNode: 'WishListComponent',
+    previousImportPath: '@spartacus/storefront',
+    newImportPath: '@spartacus/cart/wish-list/components',
+  },
+  {
+    previousNode: 'WishListItemComponent',
+    previousImportPath: '@spartacus/storefront',
+    newImportPath: '@spartacus/cart/wish-list/components',
+  },
+  {
+    previousNode: 'WishListModule',
+    previousImportPath: '@spartacus/storefront',
```

Comment:
This `WishListModule` is different with the one in `@spartacus/cart/wish-list`. It is rename to `WishListComponentsModule` in `@spartacus/cart/wish-list/components`

Reply:
created followup ticket https://github.com/SAP/spartacus/issues/15448 to review these kind of cases.